.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1-7 3,6,9,12 *"
+    # at 0:00, on the first seven days of the month, in march, june, september and december
+    # in cron, it is not possible to say "first monday of the month", so the cron is selecting
+    # the first week, and the action is only running if the day is a monday using the 
+    # is-it-monday step
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Only run on mondays
+        id: is-it-monday
+        continue-on-error: true
+        run: |
+          if [[ ! $(date +%u) -eq 1 ]]; then
+            echo 'This is not a monday.'
+            exit 1
+          fi
+
+      - name: Install cargo-release
+        if: steps.is-it-monday.outcome=='success'
+        run: cargo install cargo-release
+
+      - name: Setup release
+        if: steps.is-it-monday.outcome=='success'
+        run: |
+          git config user.name 'Bevy Auto Releaser'
+          git config user.email 'bevy@users.noreply.github.com'
+          # --workspace: updating all crates in the workspace
+          # --no-publish: do not publish to crates.io
+          # --execute: not a dry run
+          # --no-tag: do not push tag for each new version
+          # --no-push: do not push the update commits
+          # --exclude: ignore those packages
+          cargo release minor \
+            --workspace \
+            --no-publish \
+            --execute \
+            --no-tag \
+            --no-confirm \
+            --no-push \
+            --exclude ci \
+            --exclude errors \
+            --exclude bevy-ios-example
```

Comment:
Shouldn't we exclude `crevis` here, too? Currently, it gets bumped: https://github.com/mockersf/bevy/pull/62/files#diff-895f58564647a4ab76480d7ed11ff1679579391650cd98ea1329faa562abc85cL2

Reply:
I'm not sure how Cart will want to handle crevice: if vendored, it should be renamed to something like `bevy_crevice`, and the version could be changed to follow the rest of bevy, or we could freeze it to follow more closely upstream

Reply:
short term we will be vendoring crevice as bevy_crevice. Medium-to-long term, we'll either try to move back upstream, do a hard fork, or build our own thing.

Reply:
I agree that it makes sense to sync that version number with bevy